static/js/lib/queries/learning_resources.js

```diff
@@ -8,7 +8,7 @@ import {
   LR_TYPE_COURSE,
   LR_TYPE_BOOTCAMP,
   LR_TYPE_PROGRAM,
-  LR_TYPE_USERLIST_FAV
```

Comment:
Remove this from `lib/constants.js` too

Reply:
oh yeah, I meant to circle back and do that but forgot